pages/api/v1/ai-summary/refreshAISummaryOnReports.ts

```
@@ -0,0 +1,112 @@
+}): Promise<{ success: boolean; message: string; data?: any; error?: string }> => {
+	try {
+		const reportQuery = await firestore_db.collection('ai_summary_reports').where('postType', '==', postType).where('postIndex', '==', postIndex).get();
+
+		const reportCount = reportQuery.size;
+
+		if (reportCount >= 3) {
+			const { data, error } = await fetchCommentsSummaryFromPost({
+				network,
+				postId: String(postIndex),
+				postType,
+				forceRefresh: true
```

ESLint

[error] Expected object keys to be in ascending order. 'forceRefresh' should be before 'postType'.

(sort-keys)